packages/admin-api-client/package.json

```diff
@@ -0,0 +1,109 @@
+{
+  "name": "@shopify/admin-api-client",
+  "version": "0.0.0",
+  "description": "Shopify Admin API Client - A lightweight JS client to interact with Shopify's Admin API",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Shopify/shopify-api-js.git"
+  },
+  "author": "Shopify",
+  "license": "MIT",
+  "main": "./dist/umd/admin-api-client.min.js",
+  "module": "./dist/index.mjs",
+  "types": "./dist/admin-api-client.d.ts",
+  "exports": {
+    ".": {
+      "module": {
+        "types": "./dist/ts/index.d.ts",
+        "default": "./dist/index.mjs"
+      },
+      "import": {
+        "types": "./dist/ts/index.d.ts",
+        "default": "./dist/index.mjs"
+      },
+      "require": {
+        "types": "./dist/ts/index.d.ts",
+        "default": "./dist/index.js"
+      },
+      "default": "./dist/index.mjs"
+    }
+  },
+  "scripts": {
+    "lint": "eslint . --ext .js,.ts",
+    "build": "yarn run rollup",
+    "test": "jest",
+    "test:ci": "yarn test",
+    "rollup": "rollup -c --bundleConfigAsCjs",
+    "clean": "rimraf dist/*",
+    "changeset": "changeset",
+    "version": "changeset version",
+    "release": "yarn build && changeset publish"
+  },
+  "jest": {
+    "setupFilesAfterEnv": [
+      "./src/tests/setupTests.ts"
+    ],
+    "transform": {
+      ".*": "babel-jest"
+    }
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "keywords": [
+    "shopify",
+    "node",
+    "graphql",
+    "admin API"
+  ],
+  "files": [
+    "**/*.d.ts",
+    "**/*.d.ts.map",
+    "**/*.js",
+    "**/*.js.map",
+    "**/*.mjs",
+    "**/*.mjs.map",
+    "!node_modules"
+  ],
+  "dependencies": {
+    "@shopify/graphql-client": "^0.6.0"
```

Comment:
This can be updated to use `^0.7.0` - the newer version of the `graphql-client` library includes updated types and has 2 new utilities.